.github/workflows/npm.yml

```
@@ -12,14 +12,17 @@
         steps:
             - uses: actions/checkout@v4
             - uses: DeLaGuardo/setup-clojure@12.5
+              with:
+                cli: "1.11.1.1139"
```

yamllint

[error] wrong indentation: expected 18 but found 16

(indentation)